src/commands/misc/invite.js

```diff
@@ -10,9 +10,8 @@ module.exports = class Invite extends Command {
   run (message) {
     message.channel.startTyping()
     let embed = this.client.getDefaultEmbed()
-    var perm = '``MANAGE_SERVER``'
     embed.setThumbnail(this.client.user.displayAvatarURL)
-    embed.setDescription(`[Click here to invite me to your server](https://discordapp.com/oauth2/authorize?client_id=${this.client.user.id}&permissions=0&scope=bot)\nNote that you need the ${perm} permission to add bots to servers.`)
+    embed.setDescription(`[Click here to invite me to your server](https://discordapp.com/oauth2/authorize?client_id=${this.client.user.id}&permissions=0&scope=bot)\nNote that you need the \`MANAGE_SERVER\` permission to add bots to servers.`)

```

Comment:
You can use ' instead of ` now. **Suggestion:** Instead of hardcoding the invite link, you could use Client.generateInvite function.